airbyte-webapp/src/components/Table/Table.tsx

```diff
@@ -43,6 +44,9 @@ const Tr = styled.tr<{
 }>`
   background: ${({ theme, erroredRows }) => (erroredRows ? theme.dangerTransparentColor : theme.whiteColor)};
   cursor: ${({ hasClick }) => (hasClick ? "pointer" : "auto")};
+  &:hover {
+    background-color: ${({ theme }) => `${theme.greyColor0}`};
```

Comment:
The legacy colors are there to support the existing code, but the new colors are also available:

```suggestion
    background-color: ${({ theme }) => `${theme.grey50}`};
```